cli/args/flags.rs

```diff
@@ -4635,6 +4638,15 @@ fn no_check_arg() -> Arg {
     .help_heading(TYPE_CHECKING_HEADING)
 }
 
+fn connected_arg() -> Arg {
+  Arg::new("connected")
+    .long("connected")
+    .env("DENO_CONNECTED")
```

Comment:
We'll need to remove the usage of `.env` because the env file hasn't been loaded at this point. Do a search through the other code to see how it's done, but basically we need to load the env variables after loading the env file.

Reply:
I have a pending custom lint rule I need to implement for this: https://github.com/denoland/deno/issues/29886